README.md

```diff
@@ -1,3 +1,12 @@
 # loopback-next-extension-starter
 
 A starter project to create extensions for loopback-next.
+
+## Overview
+
+The extension starter will show you how to create basic extensions for LoopBack Next. The different types of extensions are: 
+
+1. Providers
+2. Decorators
+3. Components
```

Comment:
`Component` is different from the rest. It's the packaging model to bundle extensions.

Reply:
What about `Actions`?